ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/router-traffic-engineering.yml

```diff
@@ -0,0 +1,51 @@
+### Routing - Traffic Engineering ###
+router_traffic_engineering:
+  router_id:
+    ipv4: 10.0.0.1
+    ipv6: 2001:beef:cafe::1
+  segment_routing:
+    colored_tunnel_rib: true
+    policy_endpoints:
+      - address: 1.2.3.4
+        colors:
+          - value: 70810
+            binding_sid: 970810
+            description: "SRTE POLICY FOR 1.2.3.4 COLOR 70810"
+            name: "SRTE-1.2.3.4-70810"
+            path_group:
+              - preference: 180
+                explicit_null_ipv4: true
+                explicit_null_ipv6: true
```

Comment:
explicit null can be < "ipv4" | "ipv6" | "none" | "ipv4 ipv6" >. Please change these knobs as below to support multiple options:

```explicit_null: < "ipv4" | "ipv6" | "none" | "ipv4 ipv6" >```

Reply:
Let the user give these options.